base/Dockerfile

```diff
@@ -17,7 +17,8 @@ RUN apt update -y && \
         libxml2-dev \
         re2c \
         wget \
-        ca-certificates
+        ca-certificates \
+        libssh2-1-dev
```

Comment:
Let this between `libreadline-dev` and `libtiff-dev`. They are alphabetically sorted so that it is easier to identify if a package is already installed.